packages/components/src/components/ScalarMarkdown/ScalarMarkdown.vue

```diff
@@ -83,6 +83,7 @@ const html = computed(() => {
     --markdown-spacing-md: 16px;
     --markdown-line-height: 1.625;
 
+    display: contents;
```

Comment:
Are we sure this won't break things elsewhere? It makes sense but I think we're already using `ScalarMarkdown` lots of places and removing it from layout might cause unexpected side affects

Reply:
i checked as much as i could and didn't find any issues, can you give this a try?

Reply:
removed as it was indeed breaking some current styling within table